db/data.py

```diff
@@ -124,7 +125,11 @@ def submit_low_level_data(mbid, data):
         )
 
     # The data looks good, lets see about saving it
-    write_low_level(mbid, data)
+    try:
+        write_low_level(mbid, data)
+    except sqlalchemy.exc.DataError as e:
```

Comment:
Should we catch all this exception or just the specifc unicode one which we expect? Should it be caught here or in `write_low_level`?